arithmetization/src/main/java/net/consensys/linea/zktracer/module/exp/ExpOperation.java

```diff
@@ -50,7 +51,9 @@
 
 @Getter
 public class ExpOperation extends ModuleOperation {
```

Comment:
Do we need `@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = false)` here?